sdk/operationalinsights/Microsoft.Azure.Management.OperationalInsights/src/Generated/Models/Workspace.cs

```diff
@@ -51,27 +51,39 @@ public Workspace()
         /// the ID associated with the workspace.</param>
         /// <param name="sku">The SKU of the workspace.</param>
         /// <param name="retentionInDays">The workspace data retention in days.
-        /// -1 means Unlimited retention for the Unlimited Sku. 730 days is the
-        /// maximum allowed for all other Skus. </param>
+        /// Allowed values are per pricing plan. See pricing tiers
+        /// documentation for details.</param>
+        /// <param name="workspaceCapping">The daily volume cap for
+        /// ingestion.</param>
+        /// <param name="createdDate">Workspace creation date.</param>
+        /// <param name="modifiedDate">Workspace modification date.</param>
         /// <param name="publicNetworkAccessForIngestion">The network access
         /// type for accessing Log Analytics ingestion. Possible values
         /// include: 'Enabled', 'Disabled'</param>
         /// <param name="publicNetworkAccessForQuery">The network access type
         /// for accessing Log Analytics query. Possible values include:
         /// 'Enabled', 'Disabled'</param>
+        /// <param name="forceCmkForQuery">Indicates whether customer managed
+        /// storage is mandatory for query management.</param>
         /// <param name="privateLinkScopedResources">List of linked private
         /// link scope resources.</param>
+        /// <param name="features">Workspace features.</param>
         /// <param name="eTag">The ETag of the workspace.</param>
-        public Workspace(string location, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), string provisioningState = default(string), string customerId = default(string), WorkspaceSku sku = default(WorkspaceSku), int? retentionInDays = default(int?), string publicNetworkAccessForIngestion = default(string), string publicNetworkAccessForQuery = default(string), IList<PrivateLinkScopedResource> privateLinkScopedResources = default(IList<PrivateLinkScopedResource>), string eTag = default(string))
+        public Workspace(string location, string id = default(string), string name = default(string), string type = default(string), IDictionary<string, string> tags = default(IDictionary<string, string>), string provisioningState = default(string), string customerId = default(string), WorkspaceSku sku = default(WorkspaceSku), int? retentionInDays = default(int?), WorkspaceCapping workspaceCapping = default(WorkspaceCapping), string createdDate = default(string), string modifiedDate = default(string), string publicNetworkAccessForIngestion = default(string), string publicNetworkAccessForQuery = default(string), bool? forceCmkForQuery = default(bool?), IList<PrivateLinkScopedResource> privateLinkScopedResources = default(IList<PrivateLinkScopedResource>), IDictionary<string, object> features = default(IDictionary<string, object>), string eTag = default(string))
```

Comment:
There is a breaking change here, new parameters should be added at the end.
Please rearrange the parameters in the swagger file.

Reply:
moved new parameters to the end